### PR TITLE
chore: add license information to Pytorch Lightning examples

### PR DESCRIPTION
**Pull request by shiyuann**
## Checklist

- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](../docs/release-notes/README.md) for details.

[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ You may intentionally leave this section blank and remove the title.
 - [ ] User-facing API changes need the "User-facing API Change" label.
 - [ ] Release notes should be added as a separate file under `docs/release-notes/`.
 See [Release Note](../docs/release-notes/README.md) for details.
+- [ ] Licenses should be included for new code which copied and/or modified from any external code.
```
Comment (hamidzr) [inline .github/pull_request_template.md:33]: nit: IMO either of these looks better:

for **any** new code ~which~ copied and/or modified from 

or which **was** copied

File: examples/computer_vision/mnist_pl/mnist.py [hunk 2]
```diff
@@ -1,76 +1,82 @@
...
+    def test_step(self, batch, batch_idx):
+        # Here we just reuse the validation_step for testing
+        return self.validation_step(batch, batch_idx)
 
     def configure_optimizers(self):
-        optimizer = torch.optim.Adam(self.parameters(),
-                                     lr=self.hparams.lr)
+        optimizer = torch.optim.Adam(self.parameters(), lr=self.learning_rate)
         return optimizer
-
```
Comment (hamidzr) [inline examples/computer_vision/mnist_pl/mnist.py:69]: could we keep this feature? to show that it is a standalone fully functioning pytorch_lightning experiment?

File: .github/pull_request_template.md (hunk 1 above)
Comment (shiyuann) [reply .github/pull_request_template.md]: done

File: examples/computer_vision/mnist_pl/mnist.py (hunk 2 above)
Comment (shiyuann) [reply examples/computer_vision/mnist_pl/mnist.py:69]: done

File: examples/computer_vision/mnist_pl/mnist.py [hunk 3]
```diff
@@ -1,76 +1,91 @@
+# The class MNISTDataModule is modified from the Pytorch Lightning example:
```
Comment (hamidzr) [inline examples/computer_vision/mnist_pl/mnist.py:1]: do you mean `LitMNIST` ?

**Review by hamidzr — APPROVED** [review]
LGTM. thanks @shiyuann
